dashboard/src/components/searchresults/SearchResultsView.js

```diff
@@ -60,20 +60,19 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-const searchTypeMap = {
-  "Feature variant": "Feature",
-  Entity: "Entity",
-  "Label variant": "Label",
-  "Training Set variant": "TrainingSet",
-  Model: "Model",
-  "Source variant": "Source",
-  User: "User",
-  Provider: "Provider",
-};
+var searchTypeMap = {
+  '\u0004': "Feature", //feature variant
```

Comment:
What does this do?

Reply:
Since the resource types are defined as enums in the metadata.proto file, they are passed as unicode values. This converts them into their named types

Reply:
Add a comment on there for clarification

Reply:
added